src/Query/QueryBuilder.php

```diff
@@ -300,19 +300,57 @@ public function fetchFirstColumn(): array
         return $this->connection->fetchFirstColumn($this->getSQL(), $this->params, $this->paramTypes);
     }
 
+    /**
+     * Executes an SQL query (SELECT) and returns a Result.
```

Comment:
This may be misleading since it works also for `DESCRIBE`, `SHOW` and so on. Let's rephrase to:
```
Executes an SQL query that returns a Result.
```

Reply:
I'd agree with that on the `Connection` class, but AFAIK, there is no way to build anything else than a `SELECT` query with a `QueryBuilder`. So I don't think we should document an impossible use-case.

WDYT ?

Reply:
You're right, I didn't take the scope into account. Sorry for the confusion.